content/_code-samples/airgapped-wallet/py/relay-transaction.py

```diff
@@ -0,0 +1,50 @@
+from xrpl.clients import JsonRpcClient
+from xrpl.models.transactions import Payment
+from xrpl.transaction import send_reliable_submission
+
+
+def connect_node(_node):
+    """
+    Connects to a node
+    """
+
+    JSON_RPC_URL = _node
+    _client = JsonRpcClient(url=JSON_RPC_URL)
+    print("\n   ---   Connected to Node")
+    return _client
+
+
+def send_transaction(transaction_dict):
+    """
+    Connects to a node -> Send Transaction
+    Main Function to send transaction to the XRPL
+    """
+
+    client = connect_node("https://s.altnet.rippletest.net:51234/")
+    # TESTNET: "https://s.altnet.rippletest.net:51234/"
+    # MAINNET: "https://s2.ripple.com:51234/"
+
+    # Since we manually inserted the tx blob, we need to initialize it into a Payment so xrpl-py could process it
+    my_tx_signed = Payment.from_dict(transaction_dict)
+
+    tx = send_reliable_submission(transaction=my_tx_signed, client=client)
+
+    tx_hash = tx.result['hash']
+    tx_destination = tx.result['Destination']
+    tx_xrp_amount = int(tx.result['Amount']) / 1000000
+    tx_account = tx.result['Account']
+
+    print(f"\n           XRPL Explorer: https://testnet.xrpl.org/transactions/{tx_hash}"
+          f"\n         Transaction Hash: {tx_hash}"
+          f"\n  Transaction Destination: {tx_destination}"
+          f"\n           Transacted XRP: {tx_xrp_amount}"
+          f"\n              Wallet Used: {tx_account}"
+         )
+
+
+if __name__ == '__main__':
+    tx_blob = "ENTER TX BLOB HERE"
+    if tx_blob is "ENTER TX BLOB HERE":
+        print("Set tx to 'tx_blob' received from scanning the QR code generated by the airgapped wallet")
+    
+    send_transaction(tx_blob)
```

Comment:
Two suggestions: 
1. is -> ==
2. Don't attempt to use the dummy text with `send_transaction`
```suggestion
    tx_blob = "ENTER TX BLOB HERE"
    if tx_blob == "ENTER TX BLOB HERE":
        print("Set tx to 'tx_blob' received from scanning the QR code generated by the airgapped wallet")
    else:
        send_transaction(tx_blob)
```

Otherwise it crashes saying: 
```
 SyntaxWarning: "is" with a literal. Did you mean "=="?
  if tx_blob is "ENTER TX BLOB HERE":
Set tx to 'tx_blob' received from scanning the QR code generated by the airgapped wallet

   ---   Connected to Node
Traceback (most recent call last):
  File "/Users/jacksonmills/Documents/wojake-xrpl-dev-portal/xrpl-dev-portal/content/_code-samples/airgapped-wallet/py/relay-transaction.py", line 50, in <module>
    send_transaction(tx_blob)
  File "/Users/jacksonmills/Documents/wojake-xrpl-dev-portal/xrpl-dev-portal/content/_code-samples/airgapped-wallet/py/relay-transaction.py", line 28, in send_transaction
    my_tx_signed = Payment.from_dict(transaction_dict)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/xrpl/models/transactions/transaction.py", line 367, in from_dict
    return cast(Transaction, super(Transaction, cls).from_dict(value))
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/xrpl/models/base_model.py", line 92, in from_dict
    raise XRPLModelException(
xrpl.models.exceptions.XRPLModelException: E not a valid parameter for Payment```